data/data/manifests/openshift/cloud-creds-secret.yaml.template

```diff
@@ -45,4 +47,9 @@ data:
   ovirt_password: {{.CloudCreds.Ovirt.Base64encodePassword}}
   ovirt_insecure: {{.CloudCreds.Ovirt.Base64encodeInsecure}}
   ovirt_ca_bundle: {{.CloudCreds.Ovirt.Base64encodeCABundle}}
+{{- else if .CloudCreds.Nutanix}}
+  NUTANIX_ENDPOINT: {{.CloudCreds.Nutanix.Base64encodeNutanixEndpoint}}
+  NUTANIX_PORT: {{.CloudCreds.Nutanix.Base64encodeNutanixPort}}
+  NUTANIX_USER: {{.CloudCreds.Nutanix.Base64encodeNutanixUsername}}
+  NUTANIX_PASSWORD: {{.CloudCreds.Nutanix.Base64encodeNutanixPassword}}
```

Comment:
@thunderboltsid does this need to refactored in light of https://github.com/openshift/installer/pull/5631/files#diff-89d415ff2b74dae9eb1035ae2b655b8f099235198e424c4108e000601cc1dec6R218 ?

Reply:
Indeed